spec/rubocop/cop/style/redundant_freeze_spec.rb

```diff
@@ -23,6 +23,11 @@
   it_behaves_like 'immutable objects', '1.5'
   it_behaves_like 'immutable objects', ':sym'
   it_behaves_like 'immutable objects', ':""'
+  it_behaves_like 'immutable objects', "ENV['foo']"
+  it_behaves_like 'immutable objects', "'foo'.count"
```

Comment:
`String` object does not have `count` method, does this mean `String#size` or `String#length`?

Reply:
[`String` does support `count`](https://ruby-doc.org/core-2.4.0/String.html#method-i-count). Without arguments it works like `size` and `length`. It can take a parameter of a pattern to count.